### PR TITLE
hotfix: Updated prettyVc and agent configuration

### PR DESCRIPTION
**Pull request by bhavanakarwade**
### What?
- Added prettyVc height and width configurations
- Refactored organization name and organization description length validations 
- Added agent env while provisioning base wallet locally
- Applied trim validations on email_identifier field

### CODE REVIEW

File: agent.env [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+MAX_CONNECTIONS=1000
+IDLE_TIMEOUT=30000
+SESSION_ACQUIRE_TIMEOUT=2147483647
+SESSION_LIMIT=2147483647
+INMEMORY_LRU_CACHE_LIMIT=2147483647
+BCOVRIN_REGISTER_URL=http://test.bcovrin.vonx.io/register
+INDICIO_NYM_URL=https://selfserve.indiciotech.io/nym
+SCHEMA_MANAGER_CONTRACT_ADDRESS=0x289c7Bd4C7d38cC54bff370d6f9f01b74Df51b11
+FILE_SERVER_TOKEN=
+RPC_URL=https://rpc-amoy.polygon.technology
```
Comment (tipusinghaw) [inline agent.env:13]: We should use the mainnet configuration polygon as default value.

File: apps/api-gateway/src/organization/dtos/create-organization-dto.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export class CreateOrganizationDto {
   @Transform(({ value }) => trim(value))
   @IsNotEmpty({ message: 'Organization name is required.' })
   @MinLength(2, { message: 'Organization name must be at least 2 characters.' })
-  @MaxLength(50, { message: 'Organization name must be at most 50 characters.' })
+  @MaxLength(200, { message: 'Organization name must be at most 50 characters.' })
```
Comment (tipusinghaw) [inline apps/api-gateway/src/organization/dtos/create-organization-dto.ts:13]:
```suggestion
  @MaxLength(200, { message: 'Organization name must be at most 200 characters.' })
```

File: apps/api-gateway/src/organization/dtos/create-organization-dto.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ export class CreateOrganizationDto {
   @Transform(({ value }) => trim(value))
   @IsNotEmpty({ message: 'Description is required.' })
   @MinLength(2, { message: 'Description must be at least 2 characters.' })
-  @MaxLength(255, { message: 'Description must be at most 255 characters.' })
+  @MaxLength(1000, { message: 'Description must be at most 255 characters.' })
```
Comment (tipusinghaw) [inline apps/api-gateway/src/organization/dtos/create-organization-dto.ts:22]:
```suggestion
  @MaxLength(1000, { message: 'Description must be at most 1000 characters.' })
```

File: apps/api-gateway/src/organization/dtos/update-organization-dto.ts [hunk 4]
```diff
@@ -15,7 +15,7 @@ export class UpdateOrganizationDto {
     @Transform(({ value }) => trim(value))
     @IsNotEmpty({ message: 'Organization name is required.' })
     @MinLength(2, { message: 'Organization name must be at least 2 characters.' })
-    @MaxLength(50, { message: 'Organization name must be at most 50 characters.' })
+    @MaxLength(200, { message: 'Organization name must be at most 50 characters.' })
```
Comment (tipusinghaw) [inline apps/api-gateway/src/organization/dtos/update-organization-dto.ts:18]:
```suggestion
    @MaxLength(200, { message: 'Organization name must be at most 200 characters.' })
```

File: apps/api-gateway/src/organization/dtos/update-organization-dto.ts [hunk 5]
```diff
@@ -24,7 +24,7 @@ export class UpdateOrganizationDto {
     @Transform(({ value }) => trim(value))
     @IsNotEmpty({ message: 'Description is required.' })
     @MinLength(2, { message: 'Description must be at least 2 characters.' })
-    @MaxLength(255, { message: 'Description must be at most 255 characters.' })
+    @MaxLength(1000, { message: 'Description must be at most 255 characters.' })
```
Comment (tipusinghaw) [inline apps/api-gateway/src/organization/dtos/update-organization-dto.ts:27]:
```suggestion
    @MaxLength(1000, { message: 'Description must be at most 1000 characters.' })
```

**Review by GHkrishna — APPROVED** [review]
LG

File: libs/common/src/cast.helper.ts [hunk 6]
```diff
@@ -92,19 +92,16 @@ export const IsNotSQLInjection =
...
           const sqlKeywords = ['SELECT', 'INSERT', 'UPDATE', 'DELETE', 'DROP', 'UNION', 'WHERE', 'AND', 'OR'];
-        if ('string' === typeof value) {
-          // Convert the value to upper case for case-insensitive comparison
-          const upperCaseValue = value.toUpperCase();
-          // Use a regular expression to check for whole words
-          for (const keyword of sqlKeywords) {
-            const regex = new RegExp(`\\b${keyword}\\b`, 'i');
-            if (regex.test(upperCaseValue)) {
-              return false; // Value contains a SQL injection keyword
+          if ('string' === typeof value) {
```
Comment (tipusinghaw) [inline libs/common/src/cast.helper.ts:95]: Any specific reason to remove this logic `const upperCaseValue = value.toUpperCase();`

File: apps/api-gateway/src/organization/dtos/create-organization-dto.ts (hunk 2 above)
Comment (bhavanakarwade) [reply apps/api-gateway/src/organization/dtos/create-organization-dto.ts:13]: Resolved

File: apps/api-gateway/src/organization/dtos/update-organization-dto.ts (hunk 5 above)
Comment (bhavanakarwade) [reply apps/api-gateway/src/organization/dtos/update-organization-dto.ts:27]: Resolved

File: agent.env (hunk 1 above)
Comment (bhavanakarwade) [reply agent.env:13]: We are using this script for provisioning the agent locally. I think we don't need mainnet configurations for local use. What do you think?

File: libs/common/src/cast.helper.ts (hunk 6 above)
Comment (bhavanakarwade) [reply libs/common/src/cast.helper.ts:95]: Resolved

File: agent.env (hunk 1 above)
Comment (bhavanakarwade) [reply agent.env]: Resolved

**Pull request merged**